tests/src/behaviors/tx-per-second.ts

```diff
@@ -162,6 +176,64 @@ const signalTrial = async (period, playerAgents, allPlayers, channel, messagesTo
         console.log("waiting for all conductors to be listed as active", stats)
     } while (true) // TODO fix for multi-instance
 
+    // setup the signal handler for all the players so we can check
+    // if all the signals are returned
+    let receipts: { [key: string]: number; } = {};
+    for (const i in allPlayers) {
+        const conductor = allPlayers[i]
+        conductor.setSignalHandler((signal) => {
+            const me = i
+            console.log(`Received Signal for ${me}:`, signal.data.payload.signal_payload.messageData.message)
+            if (!receipts[me]) {
+                receipts[me] = 1
+            } else {
+                receipts[me] += 1
+            }
+        })
+    }
+
+    const start = Date.now()
+    await sendConcurrently(playerAgents, channel, messagesToSend)
```

Comment:
This seems like it wouldn't actually send any signals because sendConcurrently calls `send` with `signal = false`

Reply:
Hah!  wierd, wonder why it reports that it receives all the signals!!

Reply:
Oh yeah that's something I should've told you. I tried running the signal test before implementing signal tunneling and it was passing